sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -104,6 +106,21 @@ def set_ttl_seconds_after_finished(self, seconds: int):
     self.ttl_seconds_after_finished = seconds
     return self
 
+  def set_pod_disruption_budget(self, min_available: str, selector: str=None):
+    """ PodDisruptionBudget holds the number of concurrent disruptions that you allow for pipeline Pods.
+	      Controller will automatically add the selector with workflow name, if selector is empty.
+
+    Args:
+        min_available (str):  An eviction is allowed if at least "minAvailable" pods selected by 
+                              "selector" will still be available after the eviction, i.e. even in the
+	                            absence of the evicted pod.  So for example you can prevent all voluntary
+	                            evictions by specifying "100%".
+        selector (str): Label query over pods whose evictions are managed by the disruption
```

Comment:
In what scenarios do you need non-default selector?

Reply:
hmm god question, to be honest, I don't know. I just propagated the option. Maybe it would make sense to remove this as an option in the SDK? I don't have any opinon on this.

Reply:
Let's remove this option until there is a clear need.